exercises/collatz-conjecture/test_collatz-conjecture.R

```diff
@@ -0,0 +1,28 @@
+source("./collatz-conjecture.R")
+library(testthat)
+
+test_that("Input of one results in zero steps", {
```

Comment:
Haha - fair enough. To be honest I did try and re-write these a few different ways and wasn't really sure...